docs/hub/mlx-image.md

```diff
@@ -0,0 +1,89 @@
+# Using mlxim at Hugging Face
+
+[`mlxim`](https://github.com/riccardomusmeci/mlx-image) is an image models library built on Apple [MLX](https://github.com/ml-explore/mlx). It tries to replicate the great [timm](https://github.com/huggingface/pytorch-image-models) library from Ross Wightman, but for MLX models.
+
+
+## Exploring MLX on the Hub
+
+You can find `mlxim` models by filtering at the left of the [models page](https://huggingface.co/models?library=mlxim&sort=trending).
+There's also an open [mlx-vision](https://huggingface.co/mlx-vision) space for contributors converting and publishing weights for MLX format.
+
+Thanks to MLX Hugging Face Hub integration, you can load MLX models with a few lines of code. 
+
+## Installation
+
+```bash
+pip install mlx-image
+```
+## **Models**
+Models weights are available on [`mlx-vision`](https://huggingface.co/mlx-vision) space on HuggingFace.
+
+To create a model with weights:
+```python
+from mlxim.model import create_model
+
+# loading weights from HuggingFace
+model = create_model("resnet18")
+
+# loading weights from local file
+model = create_model("resnet18", weights="path/to/weights.npz")
+```
+
+To list all available models:
+```python
+from mlxim.model import list_models
+list_models()
+```
+> [!WARNING]
+> As of today (2024-03-08) mlx does not have `group` param for nn.Conv2d (e.g. `resnext`, `regnet`, `efficientnet`).
+
+## **ImageNet-1K Results**
```

Comment:
```suggestion
## ImageNet-1K Results

```

I don't think bold is necessary in headings. Also, we usually use a blank line for separation to avoid potential rendering issues.